glfw.py

```diff
@@ -480,7 +480,7 @@ class GLFWmonitor(Structure): pass
 def glfwCreateWindow(width=640, height=480, title="GLFW Window",
                      monitor=None, share=None):
     _glfw.glfwCreateWindow.restype = POINTER(GLFWwindow)
-    window = _glfw.glfwCreateWindow(width,height,title,monitor,share)
+    window = _glfw.glfwCreateWindow(width,height,title if type(title) == bytes else title.encode('utf-8'),monitor,share)
```

Comment:
Maybe you can add a dedicated line for title transformation (more readable I think)

Reply:
> Maybe you can add a dedicated line for title transformation (more readable I think)

Yeah, sure. A dedicated line could be better. I have changed the code. The set_title function has also be fixed.